app/assets/stylesheets/common/admin/admin_base.scss

```diff
@@ -74,6 +74,13 @@ $mobile-breakpoint: 700px;
 }
 
 .admin-contents {
+  --space-1: 0.25rem;
+  --space-2: calc(0.25rem * 2);
+  --space-3: calc(0.25rem * 3);
+  --space-4: calc(0.25rem * 4);
+  --space-5: calc(0.25rem * 5);
+  --space-6: calc(0.25rem * 6);
```

Comment:
Nice! Wonder if we should define this globally?

Reply:
That would be great! It would allow us to achieve consistent spacing across all our elements. I'll put up a separate PR for that.